docs/source/getting_started_guide/creating_a_basic_network.rst

```diff
@@ -28,61 +28,64 @@ Receptor configurations
   - node:
     id: foo
 
-    - control-service:
-      service: control
-      filename: /tmp/foo.sock
+  - control-service:
+    service: control
+    filename: /tmp/foo.sock
 
-    - tcp-peer:
-      address: localhost:2222
-      redial: true
+  - tcp-peer:
+    address: localhost:2222
+    redial: true
 
-    - log-level: debug
-    ...
+  - log-level: debug
+
+  ...
 
 **bar.yml**
 
 .. code-block:: yaml
 
-    ---
-    - node:
-      id: bar
+  ---
+  - node:
+    id: bar
+
+  - control-service:
+    service: control
+    filename: /tmp/bar.sock
 
-    - control-service:
-      service: control
-      filename: /tmp/bar.sock
+  - tcp-listener:
+    port: 2222
 
-    - tcp-listener:
-      port: 2222
+  - log-level: debug
 
-    - log-level: debug
-    ...
+  ...
 
- **mal.yml**
+**mal.yml**
```

Comment:
```suggestion
``mal.yml``
```

I think the convention is to use backticks (`) with filenames. If the point of bold here is to highlight, consider using a subsection.

Reply:
changed for all 3 files